### PR TITLE
age_suicide_demographics_analysis

### PR DESCRIPTION
**Pull request by 13524766**
Team,  pls kindly check this analysis on age demographics on suicide rate and approve

### CODE REVIEW

**Comment by isampe** [comment]
1. stats = pd.read_csv('C:\\Users\\USER\\Desktop\\DSPAT2\\MentalHealth\\Age-standardized suicide rates.csv')
Probably it would be better to use a relative path. For example, instead of a full path as above, it can be changed with '../raw_data/age-standardized suicide rates.csv', as long as your file structure following our repository structure (analysis, raw_data, and so on) and this python code stored in analysis folder
2.  Import line position
Referring to Python Code Guideline (https://www.python.org/dev/peps/pep-0008/#imports) that "Imports are always put at the top of the file". I suggest to move this line:
import matplotlib.pyplot as plt
import seaborn as sns
import warnings
to the top of the file

**Comment by HninPwint** [comment]
**Page Path:**
` pd.read_csv('C:\Users\USER\Desktop\DSPAT2\MentalHealth\Age-standardized suicide rates.csv')`  would not work if the code is run in the machine of team members.
Ming has created the cleaned dataset for the same file. You may take an option of using it. It is available under `/processed_data/`

_The example:_
`suicide_rates_data_filepath = os.path.join("processed_data", "crude_suicide_rates.csv")
suicide_rates_dataframe = pd.read_csv(suicide_rates_data_filepath)`

**Comment by HninPwint** [comment]
**Plot Title:**
I think the plots will look better with the **titles**.

File: analysis/question_3/analysis_hr_suicide.ipynb [hunk 1]
```diff
@@ -0,0 +1,799 @@
...
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "df= pd.read_csv('C:\\\\Users\\\\USER\\\\Desktop\\\\DSPAT2\\\\MentalHealth\\\\Human Resources.csv')"
```
Comment (MingMSDI) [inline analysis/question_3/analysis_hr_suicide.ipynb:11]: prefer to use meaningful variable name rather than "df"

**Pull request merged**